LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -4767,11 +4767,17 @@ LocFlyOverText="Retribution"
 LocPromotionPopupText="<Bullet/> If an enemy begins their turn in an adjacent tile, Retribution will trigger if that enemy tries to attack the Skirmisher.<br/><Bullet/> If an enemy does not begin their turn in an adjacent tile, then Retribution will trigger when that enemy moves into melee range.<br/>"
 
 [Battlemaster X2AbilityTemplate]
-LocFriendlyName="Battlemaster"
 ; LWOTC Needs Translation
+LocFriendlyName="Battlemaster"
 LocLongDescription="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 aim and defense from high ground, and have lower infiltration times."
 LocHelpText="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 aim and defense from high ground, and have lower infiltration times."
 LocPromotionPopupText="<Bullet/> An additional grenade can be equipped in a unique grenade-only slot.<br/><Bullet/> Gain +10 aim and defense from high ground.<br/><Bullet/> This soldier has reduced infiltration times while on missions."
+
+[SkirmisherGrapple X2AbilityTemplate]
+LocFriendlyName="Grapple"
+LocHelpText="Deploy a grappling hook to move quickly to an elevated position."
+LocLongDescription="Deploy a grappling hook to move quickly to an elevated position."
```

Comment:
I don't think you need the above lines. They're the same as vanilla WOTC, right? You only need the one below.